api/resource.rb

```diff
@@ -151,6 +154,18 @@ def validate
       end
     end
 
+    # Represents a list of documentation links.
```

Comment:
```suggestion
    # Some GCP resources have resource-level IAM policies that can give users permissions on that
    # individual resource instead of giving them wide permissions across the project/org.
    # These policies have a well-known shape across resources / products, and generally only the
    # identity (eg name, location, project) of the parent resource changes.
    # For example, https://cloud.google.com/compute/docs/reference/rest/v1/subnetworks/getIamPolicy
```

You probably don't want to actually accept this suggestion, but let's give some context on what this field does.